src/main/java/org/ow2/proactive/catalog/Application.java

```diff
@@ -91,6 +91,7 @@
     private String dataSourcePassword;
 
     public static void main(String[] args) {
+        System.setProperty("sun.java2d.cmm", "sun.java2d.cmm.kcms.KcmsServiceProvider");
```

Comment:
Could you please elaborate what does this mean?

Reply:
There is a performance degradation between java8 and pdfbox ( https://pdfbox.apache.org/2.0/getting-started.html )